### PR TITLE
Add MultiLabelsMetric

### PR DESCRIPTION
**Pull request by skpig**
### PR types
New features (issue #1247)

### PR changes
1. Add MultiLabelsMetric class in metrics/glue.py 
2. Add unit test tests/metrics/test_glue.py

### Description
1. In order to handle multi_labels classification
2. Add corresponding unit test

@ZeyuChen

### CODE REVIEW

File: paddlenlp/metrics/glue.py [hunk 1]
```diff
@@ -436,3 +437,250 @@ def name(self):
...
+            The total number of labels which is usually the number of classes
+        name (str, optional):
+            String name of the metric instance. Defaults to 'multi_labels_metric'.
+
+    Example:
+
+        .. code-block::
+
+            import paddle
+            from paddlenlp.metrics import AccuracyAndF1
```
Comment (ZeyuChen) [inline paddlenlp/metrics/glue.py]: Example code block is not correct

**Comment by skpig** [comment]
@ZeyuChen @LiuChiachi 麻烦能帮忙review一下吗

**Comment by skpig** [comment]
Thank you! If any problems during review, please feel free to @ me.

**Comment by skpig** [comment]
@ZeyuChen All checks have passed. Is it ok to be merged?

**Review by ZeyuChen — APPROVED** [review]
LGTM!
Thanks for your contributions!

**Pull request merged**